### PR TITLE
feat: Bump github.com/wapc/wapc-guest-tinygo to v.0.3.1

### PR DESCRIPTION
**Pull request by viccuad**
wapc-guest-tinygo works with tinygo >= 0.19.0.

### CODE REVIEW

**Comment by viccuad** [comment]
We will need to cut a new release after merging this, to be consumed.

**Review by ereslibre — COMMENTED** [review]
LGTM, just one comment: if you run `go mod tidy`, the `v0.3.0` dependency is removed from the `go.sum`.

**Comment by viccuad** [comment]
Doh, indeed, thanks. Removed, I suppose I staged incorrectly at first.

**Review by ereslibre — APPROVED** [review]
LGTM, thanks!

**Comment by flavio** [comment]
LGTM too

**Pull request merged**
